modules/files/files_test.go

```diff
@@ -17,10 +17,10 @@ func TestFileExists(t *testing.T) {
 	t.Parallel()
 
 	currentFile, err := filepath.Abs(os.Args[0])
-	assert.NoError(t, err)
+	require.NoError(t, err)
 
-	assert.True(t, FileExists(currentFile))
-	assert.False(t, FileExists("/not/a/real/path"))
+	require.True(t, FileExists(currentFile))
+	require.False(t, FileExists("/not/a/real/path"))
```

Comment:
Not necessary to change back, but these were actually ok to keep as `assert`. The difference between `require` and `assert` is that `require` will stop the execution of the test when the assertion fails, vs `assert` will continue execution. We were using `assert` for these checks, so that we can see all the failures at once as opposed "fix and error again."

With that said, not a big deal since we don't have a lot of assertions here, so ok to keep.